client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -44,6 +63,10 @@ export default function ReskinnedProcessingScreen( { flowName, hasPaidDomain } )
 		return () => clearTimeout( id );
 	}, [] );
 
+	if ( totalSteps === 0 ) {
```

Comment:
There's a couple places above this line where we divide by zero. Because of the JS peculiarity that `x / 0 === Infinity` I think all the logic works out ok i.e. `isComplete` gets set to `true` so no timer is ever started. But it's not super obvious how it's working when there's no steps.

I don't have an easy solution sorry. I would suggest moving this `if` statement further up, but we can't have conditionals before hooks.

Reply:
Yeah we can't have conditionals before hooks, so I think here is the best position 😅